module/proto/gravitynft/v1/attestation.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+package gravitynft.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/Gravity-Bridge/Gravity-Bridge/module/x/gravitynft/types";
+
+// NFTClaimType is the cosmos type of an event from the counterpart chain that can
+// be handled
+enum NFTClaimType {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // An unspecified claim type
+  NFT_CLAIM_TYPE_UNSPECIFIED         = 0;
+  // A claim for a SendNFTToCosmos transaction
+  NFT_CLAIM_TYPE_SEND_NFT_TO_COSMOS      = 1;
+
+  // TODO: WILL WE USE THIS?
+  // A claim for when an NFT is sent to Ethereum
+  NFT_CLAIM_TYPE_SEND_NFT_TO_ETH   = 2;
+
+  // TODO: ASK JAKUB ABOUT THIS ONE: IS THIS THE NFT-GRAVITY CONTRACT?
```

Comment:
Yes. It's emitted on contract deployment (GravityNFT.sol), so we know the starting point for the oracles to scan Eth.